uaclient/entitlements/base.py

```diff
@@ -153,6 +166,78 @@ def can_enable(self, silent: bool = False) -> bool:
             if not silent:
                 print(details)
             return False
+
+        if self.incompatible_services:
+            return self.handle_incompatible_services()
+
+        return True
+
+    def handle_incompatible_services(self):
```

Comment:
Let's do typing hints, and please add a :return: doc  at the bottom of the docstring
```suggestion
    def handle_incompatible_services(self) -> bool:
```